Tests/RevenueCatUITests/CustomerCenter/CustomerCenterViewModelTests.swift

```diff
@@ -276,7 +257,7 @@ final class CustomerCenterViewModelTests: TestCase {
 
     func testHasAnyPurchasesIsTrueWithOnlyVirtualCurrencies() async throws {
         let mockPurchases = MockCustomerCenterPurchases(
-            customerInfo: CustomerCenterViewModelTests.customerInfoWithoutSubscriptions,
+            customerInfo: CustomerCenterViewModelTests.customerInfoWithExpiredSubscriptions,
```

Comment:
I am unsure this needs to be updated since the test is testing a case when there's only VCs

Reply:
Then maybe we should create an actual `customerInfoWithoutSubscriptions` , wdyt?

Reply:
got it https://github.com/RevenueCat/purchases-ios/pull/5614/commits/4b281b6c2623058be46c05a81275945c234981a9